baybe/surrogates/utils.py

```diff
@@ -15,47 +15,6 @@
     from baybe.surrogates.base import Surrogate
 
 
-def _prepare_inputs(x: Tensor) -> Tensor:
-    """Validate and prepare the model input.
-
-    Args:
-        x: The "raw" model input.
-
-    Returns:
-        The prepared input.
-
-    Raises:
-        ValueError: If the model input is empty.
-    """
-    from baybe.utils.torch import DTypeFloatTorch
-
-    if len(x) == 0:
-        raise ValueError("The model input must be non-empty.")
-    return x.to(DTypeFloatTorch)
```

Comment:
these two functions have been deleted but I couldnt immediately spot the deleted funcitonality in the new code.

Is it not required anymore? Did I miss it?

Reply:
The two functions are no longer required because their two internal steps are already taken care of elsewhere since long:
* The took care of converting to the correct `DTypeFloatTorch`, which is anyway already handled in our `to_torch` utility that we use to transition from dataframes to torch
* The checks of input size are also no needed because 1) the training input size check is done in the BayesianRecommender (i.e. the first point of contact, where it is supposed to happen) and 2) the training target dimensionality check is not needed at all because our objectives automatically take care of the right sizes